spec/models/presentation_spec.rb

```diff
@@ -1,5 +1,109 @@
+# spec/models/presentation_spec.rb
 require 'rails_helper'
 
 RSpec.describe Presentation, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  subject { FactoryGirl.create(:presentation) }
+
+  describe 'valid model' do
+    it { is_expected.to be_valid }
```

Comment:
https://github.com/thoughtbot/shoulda-matchers

Now seems like a fantastic time to add `shoulda_matchers` to our project so that we can optimize some tests like this:

```ruby
it { is_expected.to validate_presence_of :thing }
```